index.html

```diff
@@ -1,15 +1,220 @@
-<!DOCTYPE html>
```

Comment:
looks like you changed from capital to lowercase here. not sure it matters

Reply:
This shouldn't matter, it's straight from the starter Bootstrap.